examples/gradle-kotlin-dsl/build.gradle.kts

```diff
@@ -66,11 +62,35 @@ tasks.register<proguard.gradle.ProGuardTask>("proguard") {
 
     repackageclasses("")
 
-    printmapping("build/mapping.txt")
+    printmapping("build/proguard-mapping.txt")
 
     keep("""class gradlekotlindsl.App {
                 public static void main(java.lang.String[]);
             }
     """)
 }
 
+tasks.register<proguard.gradle.ProGuardTask>("proguardWithConfigFile") {
```

Comment:
Do we need these tasks? Are they all required as a replacement for this:

```
    // Alternatively put your config in a separate file
    // configuration("config.pro")
```

Reply:
The primary motivation for adding these tasks was for integration testing here: https://github.com/Guardsquare/proguard/blob/3f6562cb449d80c4a5afe34ad5184fb708fb9436/gradle-plugin/test/proguard/gradle/GradlePluginIntegrationTest.groovy#L35-L73. 

Since the existing integration tests used the `examples`, I followed the pattern. An alternative would be to separate the test projects from the example projects.

Reply:
Ah ok, I see. Maybe leave like this for now, but with a comment separating/explaining the tasks? Since these additional tasks are not required for most ProGuard users?